pgbouncer-mixin/alerts.libsonnet

```diff
@@ -0,0 +1,92 @@
+{
+  new(this): {
+
+    groups: [
+      {
+        name: this.config.uid,
+        rules: [
+          {
+            alert: 'HighNumberClientWaitingConnections',
+            expr: |||
+              pgbouncer_pools_client_waiting_connections > %(alertsHighClientWaitingConnections)s
```

Comment:
please add filteringSelector to alerts.

Reply:
Is there a reason I would be adding filteringSelectors to these alerts? Wouldn't any instance of this alert being fired be acceptable to alert off?

Reply:
nevermind! filteringSelectors in this case is just job, so that makes sense!